ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -136,7 +135,11 @@ public void sendRequest(
     requestBuilder.headers(requestHeaders);
 
     if (data == null) {
-      requestBuilder.method(method, null);
+      if (method.equals("POST") || method.equals("PUT") || method.equals("PATCH")){
+        requestBuilder.method(method, RequestBody.create(null, new byte[0]));
```

Comment:
@qbig I was referring to the Okio `ByteString`: https://github.com/square/okio/blob/master/okio/src/main/java/okio/ByteString.java#L52

Which you can pass to `RequestBody.create()`: https://github.com/square/okhttp/blob/master/okhttp/src/main/java/com/squareup/okhttp/RequestBody.java#L60